tests/EditableCheckboxGroupFieldTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class EditableCheckboxGroupFieldTest extends SapphireTest
+{
+    public static $fixture_file = 'userforms/tests/EditableFormFieldTest.yml';
```

Comment:
* should be protected
* could you use a relative filename e.g. `EditableFormFieldTest.yml` without the full path?